### ISSUE TITLE
wA crashes when accessing a reported missing verse

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#4911]
0.10.0 (bce0ca0) 
Import the usfm3 file at https://git.door43.org/ult-pre-alignment/en_act/raw/branch/master/en_act.usfm
Verse 19:41 is reported missing on the import
Open the project in wA
Navigate to 19:41

![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/9a1d4b1b-569a-4701-bd5d-3c062f05fe76)

**Comment by PhotoNomad0** [comment]
@neutrinog - looks like this is a crash in API.js when the source verse is empty:

```
  /**
   * Lifecycle method
   * @param state
   * @param props
   * @return {*}
   */
  mapStateToProps(state, props) {
    const {tc: {contextId, targetVerseText, sourceVerse}} = props;
    if (contextId) {
      const {reference: {chapter, verse}} = contextId;
      const targetTokens = Lexer.tokenize(removeUsfmMarkers(targetVerseText));
      const sourceTokens = tokenizeVerseObjects(sourceVerse.verseObjects);

```

**Comment by da1nerd** [comment]
Thanks for the write up @PhotoNomad0 . There's a fix for this now.

**Comment by da1nerd** [comment]
@PhotoNomad0 why did you connect #4916 to this issue? This was fixed in https://github.com/translationCoreApps/wordAlignment/pull/70

**Comment by PhotoNomad0** [comment]
> @PhotoNomad0 why did you connect #4916 to this issue? This was fixed in translationCoreApps/wordAlignment#70

@neutrinog - that is a fix for the original USFM3 export bug that exposed this bug.  Maybe could have waited for another issue to be written, but rushing to get a 0.10.1 release out with these two fixes.

**Comment by cckozie** [comment]
Works in 1.0.0 (0fe0fb6) and 0.10.0 (bce0ca0)
